tests/epyccel/test_epyccel_sets.py

```diff
@@ -364,3 +335,54 @@ def copy_from_arg2(a : 'set[float]'):
     python_result = copy_from_arg2(a)
     assert isinstance(python_result, type(pyccel_result))
     assert python_result == pyccel_result
+
+@pytest.fixture( params=[
+        pytest.param("fortran", marks = [
+            pytest.mark.skip(reason="pop methods not implemented in fortran"),
```

Comment:
Can we give this a more generic reason so it can be reused for future methods

```suggestion
            pytest.mark.skip(reason="set methods not implemented in fortran"),
```

Reply:
Fixed : 7ef1b00